README.md

```diff
@@ -347,6 +347,13 @@ $ make -C out
 $ ./out/Debug/run-tests
 ```
 
+Some tests are timing sensitive. Relaxing test timeouts may be necessary
+on slow or overloaded machines:
+
+```bash
+$ env UV_TIMEOUT_MULTIPLIER=2 ./out/Debug/run-tests  # 10s instead of 5s
```

Comment:
One tiny suggestion: what about naming it `UV_TEST_TIMEOUT_MULTIPLIER` so it's clear it's only for testing purposes? I'm fine either way though.

Reply:
Updated. I left out the TEST_ part in my initial commit because it's another five characters but I'm fine either way.